engine/src/flutter/lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -4041,6 +4044,115 @@ void _testTables() {
   semantics().semanticsEnabled = false;
 }
 
+void _testMenus() {
+  test('nodes with menu role', () {
```

Comment:
Can you add a test to check aria-haspopup and aria-owns

Reply:
Done!